qa/suites/upgrade/telemetry-upgrade/reef-x/1-tasks.yaml

```diff
@@ -45,10 +62,16 @@ tasks:
             - ceph versions | jq -e '.overall | keys' | grep $sha1
     - print: "**** done end upgrade..."
 
+    - print: "**** done set require-osd-release ..."
+    - cephadm.shell:
+        env: [sha1]
+        mon.a:
+            - ceph osd require-osd-release squid
```

Comment:
What are we avoiding by setting this?

Reply:
We avoid a warning like this in the cluster log:

```
cluster [WRN] Health check failed: all OSDs are running squid or later but require_osd_release < squid (OSD_UPGRADE_FINISHED)
```